src/auth/LoginForm.tsx

```diff
@@ -100,18 +100,16 @@ const LoginForm = ({
       history.replaceState(null, '', loginPath + window.location.search);
     }
 
+    authState._featheryHosted = _featheryHosted;
```

Comment:
this never should have been in the setAuthId fn definition....that is completely my bad for not realizing the indentation